frontends/mit-open/webpack.config.js

```diff
@@ -13,15 +13,42 @@ const TsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin")
 const HtmlWebpackPlugin = require("html-webpack-plugin")
 const CopyPlugin = require("copy-webpack-plugin")
 const ReactRefreshWebpackPlugin = require("@pmmmwh/react-refresh-webpack-plugin")
+const { cleanEnv, str, bool, port } = require("envalid")
 
 const {
   NODE_ENV,
   ENVIRONMENT,
   PORT,
   MITOPEN_AXIOS_BASE_PATH,
-  API_BASE_URL,
+  API_DEV_PROXY_BASE_URL,
   WEBPACK_ANALYZE,
-} = process.env
+} = cleanEnv(process.env, {
+  ENVIRONMENT: str({
+    choices: ["local", "docker", "production"],
+    default: "production",
+  }),
+  NODE_ENV: str({
+    choices: ["development", "production", "test"],
+    default: "production",
+  }),
+  PORT: port({
+    desc: "Port to run the development server on",
+    default: 8062,
+  }),
+  MITOPEN_AXIOS_BASE_PATH: str({
+    desc: "Base URL for API requests",
+    devDefault: "",
```

Comment:
Since this is set as a dev default, we'll get an error in production if it's not set. This is desirable.

The error looks like:
```
================================
 Missing environment variables:
    MITOPEN_AXIOS_BASE_PATH: Base URL for API requests
================================

 Exiting with error code 1
```